src/Parser/Core/Modules/Features/BaseHealerStatWeights.js

```diff
@@ -170,7 +173,7 @@ class BaseHealerStatWeights extends Analyzer {
     }
     const currInt = this.statTracker.currentIntellectRating;
     // noinspection PointlessArithmeticExpressionJS
-    const healIncreaseFromOneInt = (1 * ARMOR_INT_MULTIPLIER) / currInt;
+    const healIncreaseFromOneInt = ARMOR_INT_MULTIPLIER / currInt;
```

Comment:
That was on purpose since it's more technically right. :( I guess `ARMOR_INT_MULTIPLIER` would be 0.05 and we do `(1 + ARMOR_INT_MULTIPLIER)`

Reply:
I mean they're equally technically right.